force-app/main/default/classes/customCompensationTableController.cls.dup

```diff
@@ -0,0 +1,51 @@
+public class customCompensationTableController {
+    @AuraEnabled
+    public static Compensation__c[] fetchCompensation(){
+        Compensation__c[] compReturn = new Compensation__c[]{};
+        compReturn =  [SELECT Contact__r.Id, Contact__r.Name, Contact__r.Birthdate, Job_Category__c, RecordType.Name, Amount__c, Location__c, 
+                       Office__c, Submitted__c, Status__c FROM Compensation__c];
+        return compReturn ;
+    }
+
+    @AuraEnabled
+    public static Compensation__c[] fetchCompensationType(String sRecordType, String sRecordSubmitted){
+      System.debug('The type is: ' + sRecordType);
+	  System.debug('The status is : ' + sRecordSubmitted);
+       // Return all compensation when no search term
+        sRecordType = sRecordType.trim();
+        sRecordSubmitted = String.valueOf(sRecordSubmitted);
+         
+        //Prepare query paramters
+        sRecordType = '%'+ sRecordType +'%';
+ 		//If type have is not empty and status is
+        if(sRecordType != '' && sRecordSubmitted == ''){
+            //Execute search query
+            return [SELECT Id, Contact__r.Id, Contact__r.Name, Contact__r.Birthdate, Job_Category__c, RecordType.Name, Amount__c, 
+            Location__c, Office__c, Submitted__c, Status__c FROM Compensation__c WHERE RecordType.Name LIKE :sRecordType];
+        }
+        //If type have is empty status is not
+        if(sRecordType == '' && sRecordSubmitted != ''){
+            //Execute search query
+            return [SELECT Contact__r.Id, Contact__r.Name, Contact__r.Birthdate, Job_Category__c, RecordType.Name, Amount__c, 
+            Location__c, Office__c, Submitted__c, Status__c FROM Compensation__c WHERE Submitted__c = :Boolean.valueOf(sRecordSubmitted)];
+         }
+         //If both of them arent empty
+        if(sRecordType != '' && sRecordSubmitted != ''){
```

Comment:
you could use the isBlank method instead of comparing against " ". i.e: isBlank(inputString)
Returns true if the specified String is white space, empty (''), or null; otherwise, returns false.
Signature
public static Boolean isBlank(String inputString)

https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_string.htm

Reply:
Ok, thanks i will change that